src/tools/s2wasm.cpp

```diff
@@ -33,6 +34,7 @@ int main(int argc, const char *argv[]) {
   bool generateEmscriptenGlue = false;
   std::string startFunction;
   std::vector<std::string> archiveLibraries;
+  bool validate = false;
```

Comment:
Sure. But I'm guessing it's just a few simple errors, so seems simpler to fix them here? But if it's more than that or you're not sure, then I'm ok with a followup to make it mandatory.